docs/tutorial/shell.md

```diff
@@ -64,6 +64,57 @@ let x = 1; (\a a + b)(x)
 a x
 ```
 
+It is also possible to navigate through the frames by using the `/up` and
+`/down`, or `/u` and `/d` which are shorthand for the commands.
+
+Running the following script:
+```arrai
+let a = {"a": {"b": {"c": 1}}};
+let x = "b";
+let y = "d";
+let d = (
+    \z
+    let b = a("a");
+    b(z)(y)
+);
+d(x)
+```
+
+Will drop into this shell.
+
+```bash
+INFO[0000] Call: no return values from set {c: 1}
+
+test.arrai:7:5:
+    let b = a("a");
+    b(z)(y)
+
+test.arrai:9:1:
+);
+d(x)
+
+@> <tab>
+// a  b  x  y  z
+@> /u
+2020-06-09T12:50:58.306199+10:00 INFO Stack: 0
```

Comment:
Very cool! Maybe explain what the stack number means. Is zero the top or bottom? Which way is up or down?